packages/block-editor/src/components/block-pattern-setup/setup-toolbar.js

```diff
@@ -18,7 +18,7 @@ import { VIEWMODES } from './constants';
 const Actions = ( { onBlockPatternSelect } ) => (
 	<div className="block-editor-block-pattern-setup__actions">
 		<Button variant="primary" onClick={ onBlockPatternSelect }>
-			{ __( 'Choose' ) }
+			{ __( 'Choose pattern' ) }
```

Comment:
Consistency with other places in the editor, where "Choose" is followed by the thing we choose (example: `Choose variation`)